data/json/overmap/overmap_mutable/farm_horse_mutable.json

```
@@ -4,7 +4,7 @@
     "id": "Horse Farm Mutable",
     "subtype": "mutable",
     "locations": [ "forest_without_trail", "field", "open_air" ],
-    "city_distance": [ 5, 40 ],
+    "city_distance": [ 5, 120 ],
```

```suggestion
    "city_distance": [ 5, 120 ],
```

Does -1 work for infinite distance in city distance?  If it does we should use that for things that should be anywhere  in rural regions.

---

Yeah -1 works fine

---

On further thought, I agree on these having an infinite range.  For consistency sake.